src/routes/AgreementViewRoute.js

```diff
@@ -44,6 +44,7 @@ class AgreementViewRoute extends React.Component {
       records: 'results',
       recordsRequired: '%{agreementEresourcesCount}',
     },
+    filterPath: { initialValue: 'current' },
```

Comment:
`eresourcesFilterPath` might be more self-explanatory and less likely to clash with future filters we need to implement.